src/components/wallet-card.tsx

```diff
@@ -83,64 +68,67 @@ export const WalletCard = ({address}: BalanceProps) => {
-        <Text t12 style={page.name} ellipsizeMode="tail" numberOfLines={1}>
+      <View
+        style={[styles.topNav, !wallet.mnemonicSaved && styles.marginBottom]}>
+        <Text t12 style={styles.name} ellipsizeMode="tail" numberOfLines={1}>
           {wallet.name || 'name'}
         </Text>
-        <IconButton onPress={onPressQR} style={page.qrButton}>
-          <QRCode color={LIGHT_GRAPHIC_BASE_3} />
+        <IconButton onPress={onPressQR} style={styles.qrButton}>
+          <Icon name="qr_code" color={Color.graphicBase3} />
         </IconButton>
-        <CopyButton style={page.copyButton} value={wallet.address}>
-          <Text t14 style={page.address}>
+        <CopyButton style={styles.copyButton} value={wallet.address}>
+          <Text t14 color={Color.textBase3}>
             {formattedAddress}
           </Text>
-          <Copy color={LIGHT_GRAPHIC_BASE_3} style={page.marginLeft} />
+          <Icon
+            name="copy"
+            color={Color.graphicBase3}
+            style={styles.marginLeft}
+          />
         </CopyButton>
       </View>
       {!wallet.mnemonicSaved && (
-        <IconButton onPress={onClickBackup} style={page.cacheButton}>
-          <Text clean style={page.cacheText}>
-            Without backup
-          </Text>
+        <IconButton onPress={onClickBackup} style={styles.cacheButton}>
+          <Text
+            t15
+            i18n={I18N.walletCardWithoutBackup}
+            color={Color.textBase3}
+          />
         </IconButton>
       )}
-      <Text t0 style={page.balance} numberOfLines={1} adjustsFontSizeToFit>
+      <Text t0 color={Color.textBase3} numberOfLines={1} adjustsFontSizeToFit>
         {cleanNumber(balance.toFixed(2))} ISLM
       </Text>
       <Spacer />
-      <View style={page.buttonsContainer}>
-        <View style={page.button}>
+      <View style={styles.buttonsContainer}>
+        <View style={styles.button}>
           {IS_IOS && <BlurView action="sent" cardState={cardState} />}
-          <IconButton style={page.spacer} onPress={onPressSend}>
-            <ArrowSend color={LIGHT_GRAPHIC_BASE_3} />
-            <Text clean style={page.buttonText}>
-              Send
-            </Text>
+          <IconButton style={styles.spacer} onPress={onPressSend}>
+            <Icon name="arrow_send" color={Color.graphicBase3} />
+            <Text i18n={I18N.walletCardSend} color={Color.textBase3} />
           </IconButton>
         </View>
-        <View style={page.button}>
+        <View style={styles.button}>
           {IS_IOS && <BlurView action="receive" cardState={cardState} />}
-          <IconButton style={page.spacer} onPress={onPressQR}>
-            <ArrowReceive color={LIGHT_GRAPHIC_BASE_3} />
-            <Text clean style={page.buttonText}>
-              Receive
-            </Text>
+          <IconButton style={styles.spacer} onPress={onPressQR}>
+            <Icon name="arrow_receive" color={Color.graphicBase3} />
```

Comment:
Determine the size of the icon using the prop

Reply:
done